### PR TITLE
Fix small issues

### PR DESCRIPTION
**Pull request by reinier-millo**
- Add notification on AI key validation
- Prevent keep polling the backend after leaving result page

### CODE REVIEW

File: client/app/components/keySettings/baseKeySettings.tsx [hunk 1]
```diff
@@ -40,10 +42,23 @@ export default function BaseKeysSettings({
     const isValid = await isValidAIModelKey(provider, keyValue);
     if (!isValid) {
       setKeySaved(false);
+      toast({
+        title: "Invalid key",
```
Comment (belindamo) [inline client/app/components/keySettings/baseKeySettings.tsx:46]: Change description "Please provide a working key for {provider}"

Comment (belindamo) [reply client/app/components/keySettings/baseKeySettings.tsx:46]: Recommendation: up the duration of toasts to 8000

File: client/app/lib/hooks/useEvalResults.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
-import { useEffect, useRef, useState } from 'react';
-import { getEvalRun } from '../../utils/getEvalRun';
-import { IEvalRunResponse } from '../types';
+import { useEffect, useRef, useState } from "react";
+import { getEvalRun } from "../../utils/getEvalRun";
+import { IEvalRunResponse } from "../types";
 
-
-const FINISHED_STATUS = ['Failed', 'Finished'];
+const FINISHED_STATUS = ["Failed", "Finished"];
```
Comment (belindamo) [inline client/app/lib/hooks/useEvalResults.ts:5]: "Failed", "Done"

File: client/app/lib/hooks/useEvalResults.ts
Comment (belindamo) [inline client/app/lib/hooks/useEvalResults.ts:1]: Just a heads up that in https://github.com/openevals/evals/pull/23 I take out the use of useEvalResults...but I think it makes a lot of sense to keep it incorporated!

File: backend/alembic/versions/5b1952a1c551_add_status_field_to_evalrun.py [hunk 3]
```diff
@@ -5,28 +5,37 @@
...
 
-from alembic import op
 import sqlalchemy as sa
-
+from alembic import op
 
 # revision identifiers, used by Alembic.
-revision: str = '5b1952a1c551'
-down_revision: Union[str, None] = '7442b38ebae6'
+revision: str = "5b1952a1c551"
```
Comment (justinlinw) [inline backend/alembic/versions/5b1952a1c551_add_status_field_to_evalrun.py:15]: Do these revisions have the change from `origin/development`(https://github.com/openevals/evals/pull/24) in place? If not, can we re-generate these revision so it properly includes `ae2a8d3bd293` in alembic history?

Comment (reinier-millo) [reply backend/alembic/versions/5b1952a1c551_add_status_field_to_evalrun.py:15]: That revision is in place.

**Review by justinlinw — APPROVED** [review]

**Pull request merged**